### PR TITLE
316535 Revamp Javascript sample for embedding dashboard in iframe

### PR DESCRIPTION
**Pull request by anuabarnabsf4019**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+ ![Dashboard_Iframe_js view](https://github.com/boldbi/iframe-dashboard-javascript-sample/assets/129487075/645a8740-c404-4893-b914-da531a0b4bc1)
+
+## Prerequisites
+
+ * [Node.js](https://nodejs.org/en/)
+ * [Visual Studio Code](https://code.visualstudio.com/download)
+
+ > **NOTE:** Node.js v14.16 to v18.16 are supported.
+
+#### Help link
```
Comment (vinothkumar1732) [inline README.md]: Remove this Help link topic.

File: scripts/Properties.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+debugger
```
Comment (vinothkumar1732) [inline scripts/Properties.js]: Remove this debugger

File: scripts/Properties.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+var rootUrl = "", siteIdentifier = "", email = "", environment = "", EmbedSecret = "";
```
Comment (vinothkumar1732) [inline scripts/Properties.js]: use camelcasing for embedSecret.

File: scripts/Properties.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+var baseUrl = "";
+var dashboardServerApiUrl = "";
+var dataServiceUrl = "";
+var designerServiceUrl = "";
+var designerRootUrl = "";
+var intermediateDbStatus = false;
```
Comment (vinothkumar1732) [inline scripts/Properties.js]: Is this variable using? Cross verify other variables too.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anuabarnabsf4019) [reply README.md]: removed

File: scripts/Properties.js (hunk 2 above)
Comment (anuabarnabsf4019) [reply scripts/Properties.js]: removed

File: scripts/Properties.js (hunk 3 above)
Comment (anuabarnabsf4019) [reply scripts/Properties.js]: changed variable name

File: scripts/Properties.js (hunk 4 above)
Comment (anuabarnabsf4019) [reply scripts/Properties.js]: Removed unwanted variables

File: README.md [hunk 5]
```diff
@@ -1,2 +1,89 @@
-# iframe-dashboard-javascript-sample
-This application aims to demonstrate how to embed the dashboard module into your JavaScript-based web application using an iFrame
+# Dashboard Iframe Embedding with JavaScript
+
+  This application aims to demonstrate how to embed the Bold BI dashboard into JavaScript-based web application using an iFrame by integrating the Dashboards Listing, Designing and Viewing modules.
```
Comment (M-A32) [inline README.md]: This application aims to demonstrate how to embed the Bold BI dashboard into a JavaScript-based web application using an iFrame by integrating the Dashboards Listing, Designing, and Viewing modules.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,89 @@
...
+* To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+* Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/iframe-dashboard-javascript-sample/tree/master) within the application. Please ensure that you have placed it in the application as shown in the following image.
+  
+    ![EmbedConfig image](https://github.com/boldbi/iframe-dashboard-javascript-sample/assets/129487075/15db9675-ab67-4478-a89e-4786986a564d)
+
+## Run a Sample Using Command Line Interface
```
Comment (M-A32) [inline README.md]: ## Run a Sample Using the Command Line Interface

**Review by M-A32 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (anuabarnabsf4019) [reply README.md]: updated

File: README.md (hunk 6 above)
Comment (anuabarnabsf4019) [reply README.md]: updated

**Review by vinothkumar1732 — APPROVED** [review]
Code changes are fine.

**Pull request merged**